src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

```diff
@@ -17,10 +17,7 @@ The UI for external services is a great place to analyze a single service along
 
 ## How to find the external services feature [#where-externals]
 
-The external services feature is available from the left navigation pane once you select a service in Explorer. Do one of the following:
-
-* New Relic Explorer: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Explorer > (select an app) > Monitor > External services**.
-* APM: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services > (select an app) > Monitor > External services**.
+The external services feature is available from the left navigation pane once you select a service in Explorer. You can open it by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & Services**. Then, click your app and, under the **Monitor** section, click **External services**.
```

Comment:
```suggestion
To access the external services feature, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & Services**. Select an app, then under the **Monitor** section click **External services**.
```

Reply:
@nbaenam , I was told we bold the entire UI path, including the `>`.

Reply:
Alba told me the opposite, that's why I changed it... I'm going to change it again. Thank you for letting me know.